docs/update_and_migration/from_4.3/update_from_4.3_new_commerce.md

```diff
@@ -0,0 +1,341 @@
+---
+description: Update procedure to v4.4 for people who don't use Commerce packages and can remove them.
+---
+# Update with new Commerce packages
+
+This update procedure applies if you have a v4.3 installation, and you do not use Commerce packages.
+
+## Update from v4.3.x to v4.3.latest
+
+Before you update to v4.4, you need to go through the following steps to update to the latest maintenance release of v4.3 (v[[= latest_tag_4_3 =]]).
+
+### Update the application to v4.3.latest
+
+Run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:[[= latest_tag_4_3 =]] --with-all-dependencies --no-scripts
+    ```
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer require ibexa/experience:[[= latest_tag_4_3 =]] --with-all-dependencies --no-scripts
+    ```
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer require ibexa/commerce:[[= latest_tag_4_3 =]] --with-all-dependencies --no-scripts
+    ```
+
+## Remove deprecated Field Types
+
+By default, every v4.3 installation has a set of built-in Content Types.
+Some of them use Field Types deprecated in v4.4, and they need to be removed manually.
```

Comment:
```suggestion
Some of them use Field Types deprecated in v4.4, which need to be removed manually.
```
otherwise it sounds like removing the Content Types, for Fields